.github/workflows/release-test.yml

```diff
@@ -52,3 +52,93 @@ jobs:
           name: dist-artifacts
           path: dist/
           retention-days: 1
+
+      # Linux package uploads to Google Artifact Registry
+
+      # google-github-actions/auth@v2 was used above to install
+      # gcloud sdk. google-github-actions/auth was used as well
+      # but we need to re-authenticate to upload packages to the
+      # correct project.
+      - name: "Authenticate to Google Cloud"
+        uses: "google-github-actions/auth@v2"
+        with:
+          credentials_json: "${{ secrets.ORG_BINDPLANE_PUBLIC_LINUX_REPO_JSON_KEY }}"
+
+      - name: Upload Bindplane Linux AMD64 Deb Package
+        run: |
+          gcloud \
+            --project bindplane-repository \
+            artifacts apt upload devel-deb \
+            --location us-central1 \
+            --source dist/observiq-otel-collector_${{ env.VERSION }}_linux_amd64.deb
```

Comment:
The upload commands use the version passed in by the user. This is different than the release workflow.